monix-reactive/shared/src/main/scala/monix/reactive/internal/operators/IntersperseObservable.scala

```diff
@@ -0,0 +1,56 @@
+package monix.reactive.internal.operators
+
+import monix.execution.Ack.Continue
+import monix.execution.atomic.AtomicBoolean
+import monix.execution.{Ack, Cancelable}
+import monix.reactive.Observable
+import monix.reactive.observers.Subscriber
+
+import scala.concurrent.Future
+
+/**
+  * Created by omainegra on 5/29/17.
```

Comment:
As a general rule, I abstained from adding names in source files - you can add yourself to the AUTHORS file.